modules/face/samples/sample_train_landmark_detector.cpp

```diff
@@ -0,0 +1,99 @@
+#include "opencv2/face.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgcodecs.hpp"
+
+using namespace std;
+using namespace cv;
+using namespace cv::face;
+
+int main(int argc,char** argv){
+    //Give the path to the directory containing all the files containing data
+   CommandLineParser parser(argc, argv,
+        "{ help h usage ? |      | give the following arguments in following format }"
+        "{ annotations a  |.     | (required) path to annotations txt file [example - /data/annotations.txt] }"
+        "{ config c       |      | (required) path to configuration xml file containing parameters for training.[ example - /data/config.xml] }"
+        "{ model m        |      | (required) path to configuration xml file containing parameters for training.[ example - /data/model.dat] }"
+        "{ width w        |  460 | The width which you want all images to get to scale the annotations. large images are slow to process [default = 460] }"
+        "{ height h       |  460 | The height which you want all images to get to scale the annotations. large images are slow to process [default = 460] }"
+        "{ face_cascade f |      | Path to the face cascade xml file which you want to use as a detector}"
+    );
+    //Read in the input arguments
+    if (parser.has("help")){
+        parser.printMessage();
+        cerr << "TIP: Use absolute paths to avoid any problems with the software!" << endl;
```

Comment:
Same overall, the cerr messages are just a copy paste from somewhere, they need to be meaningful.